ppr-ui/src/components/mhrRegistration/HomeLocation/HomeLandOwnership.vue

```diff
@@ -52,15 +138,28 @@ export default defineComponent({
 
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
-.mhr-home-land-ownership::v-deep {
 
-  .ownership-checkbox {
-    label {
-      line-height: 24px;
-    }
-    .v-input__slot {
-      align-items: flex-start;
-    }
-  }
+.yes-radio {
+  width: 44%;
+  margin-right: 24px !important;
+  background-color: rgba(0, 0, 0, 0.06);
+  height: 60px;
+  padding-left: 10px;
+}
+
+.no-radio {
+  width: 44%;
+  background-color: rgba(0, 0, 0, 0.06);
+  height: 60px;
+  padding: 10px;
+  margin-right: 0px !important;
+}
```

Comment:
There might be some generic classes to already utilize for these radio button pairs, that match our designs. 
If not, no worries.

Reply:
@RuoxuanPengBC

Reply:
I don't think there are suitable existing radio button class available